src/pages/Filing/FilingApp/FieldEntry.tsx

```diff
@@ -46,27 +78,54 @@ function FieldEntry({ fieldObject }: FieldEntryProperties): JSX.Element {
         <Heading type='4'>{validationName}</Heading>
         <Markdown>{validationDescription}</Markdown>
       </div>
-      <Table
-        className='w-full max-w-full table-auto'
-        columns={columns}
-        // rows={[...Array.from({ length: 100 }).keys()].map((item, index) => [
-        //   index + 10_000,
-        //   '4234000O91BZ2SUPERCALIFRAGILISTICEXPIALI45CHARS',
-        //   '4234000O91BZ2SUPERCALIFRAGILISTICEXPIALI45CHARS',
-        // ])}
-        // rows={[
-        //   [
-        //     'Row 1, Column 1',
-        //     'Row 1, Column 123456789TESTBANK123C2  123456789TESTBANK123C2  123456789TESTBANK123C2 123456789TESTBANK123C2 123456789TESTBANK123C2 123456789TESTBANK123C2',
-        //     'Row 1, Column 3 123456789TESTBANK123C2123456789TESTBANK123C2123456789TESTBANK123C2123456789TESTBANK123C2',
-        //   ],
-        //   ['Row 2, Column 1', 'Row 2, Column 2', 'Row 2, Column 3'],
-        //   ['Row 3, Column 1', 'Row 3, Column 2', 'Row 3, Column 3'],
-        // ]}
-        // @ts-expect-error TypeScript error needs to be resolved within DSR
-        rows={rows}
-        isScrollableHorizontal
-      />
+      <div className='mb-[0.9375rem]'>
+        <Table
+          className='w-full max-w-full table-auto'
+          columns={columns}
+          // rows={[...Array.from({ length: 100 }).keys()].map((item, index) => [
+          //   index + 10_000,
+          //   '4234000O91BZ2SUPERCALIFRAGILISTICEXPIALI45CHARS',
+          //   '4234000O91BZ2SUPERCALIFRAGILISTICEXPIALI45CHARS',
+          // ])}
+          // rows={[
+          //   [
+          //     'Row 1, Column 1',
+          //     'Row 1, Column 123456789TESTBANK123C2  123456789TESTBANK123C2  123456789TESTBANK123C2 123456789TESTBANK123C2 123456789TESTBANK123C2 123456789TESTBANK123C2',
+          //     'Row 1, Column 3 123456789TESTBANK123C2123456789TESTBANK123C2123456789TESTBANK123C2123456789TESTBANK123C2',
+          //   ],
+          //   ['Row 2, Column 1', 'Row 2, Column 2', 'Row 2, Column 3'],
+          //   ['Row 3, Column 1', 'Row 3, Column 2', 'Row 3, Column 3'],
+          // ]}
+          // @ts-expect-error TypeScript error needs to be resolved within DSR
+          // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+          rows={itemsToShow}
+          isScrollableHorizontal
+        />
+        {/* NOTE: Table used to create space */}
```

Comment:
Can we expand this comment a bit for future developers (why and under what circumstance is it needed) and/or make a dedicated boolean for `showPagination && ITEMS_PER_PAGE > itemsToShow.length ` that's something like `isHiddenTableAdded`?